public/src/components/App.jsx

```diff
@@ -1,28 +0,0 @@
-// This component handles the App template used on every page.
-import React,{PropTypes} from 'react';
-import Header from './templates/header.jsx';
-import {connect} from 'react-redux';
-class App extends React.Component {
-  render() {
-    return (
-      <div>
-          <Header
-            logedIn={this.props.userLoged}
-            path = {this.props.location.pathname}
-            />
-          {this.props.children}
-      </div>
-    );
-  }
-}
-
-
-const mapStateToProps = (store) => {
-  return {userLoged: store.signin}
```

Comment:
In ES6, fat arrow with parentheses is an implicit return, so this can just be written as 
(store) => (
{
userLogged: store.signin, ... }) without need for '{return...}'. This is the case for all functions, but can be immediately applied to all mapStateToProps and mapDispatchToProps.